diff --git a/absl/time/CMakeLists.txt b/absl/time/CMakeLists.txt
index 5909832..087bb6a 100644
--- a/absl/time/CMakeLists.txt
+++ b/absl/time/CMakeLists.txt
@@ -82,10 +82,12 @@ absl_cc_library(
     "internal/cctz/src/zone_info_source.cc"
   COPTS
     ${ABSL_DEFAULT_COPTS}
-  DEPS
-    $<$<PLATFORM_ID:Darwin>:${CoreFoundation}>
 )
 
+if(APPLE)
+  target_link_options(time_zone PUBLIC -framework CoreFoundation)
+endif()
+
 absl_cc_library(
   NAME
     test_util
