--- ./absl/copts/copts.py.oirg	2020-07-30 12:57:32.000000000 +0200
+++ ./absl/copts/copts.py	2020-07-30 12:57:51.000000000 +0200
@@ -182,10 +182,7 @@
         "/wd4996",  # use of deprecated symbol
         "/DNOMINMAX",  # disable the min() and max() macros from <windows.h>
     ],
-    "ABSL_MSVC_LINKOPTS": [
-        # Object file doesn't export any previously undefined symbols
-        "-ignore:4221",
-    ],
+    "ABSL_MSVC_LINKOPTS": [],
     # "HWAES" is an abbreviation for "hardware AES" (AES - Advanced Encryption
     # Standard). These flags are used for detecting whether or not the target
     # architecture has hardware support for AES instructions which can be used
--- ./absl/copts/GENERATED_AbseilCopts.cmake.orig	2020-07-30 12:57:39.000000000 +0200
+++ ./absl/copts/GENERATED_AbseilCopts.cmake	2020-07-30 12:58:31.000000000 +0200
@@ -184,10 +184,6 @@
     "/wd4800"
 )
 
-list(APPEND ABSL_MSVC_LINKOPTS
-    "-ignore:4221"
-)
-
 list(APPEND ABSL_MSVC_TEST_FLAGS
     "/wd4018"
     "/wd4101"
