From b6c0b4ffa8aa2ed4146e2c9ca9b0406e2da4f1d3 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 1 Jul 2022 13:37:17 +0200
Subject: [PATCH 04/11] add missing osx workaround

---
 absl/debugging/internal/examine_stack.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/absl/debugging/internal/examine_stack.cc b/absl/debugging/internal/examine_stack.cc
index 57863228..91862aed 100644
--- a/absl/debugging/internal/examine_stack.cc
+++ b/absl/debugging/internal/examine_stack.cc
@@ -26,6 +26,10 @@
 #include <sys/mman.h>
 #endif
 
+#if defined(MAP_ANON) && !defined(MAP_ANONYMOUS)
+#define MAP_ANONYMOUS MAP_ANON
+#endif
+
 #if defined(__linux__) || defined(__APPLE__)
 #include <sys/ucontext.h>
 #endif
