From 0a382d9756dfe67ca2972b4010063c306b78c014 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Wed, 2 Mar 2022 08:12:13 +1100
Subject: [PATCH 5/5] add some missing internal dll targets

---
 CMake/AbseilDll.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMake/AbseilDll.cmake b/CMake/AbseilDll.cmake
index 2c7f162..60fe7ce 100644
--- a/CMake/AbseilDll.cmake
+++ b/CMake/AbseilDll.cmake
@@ -347,6 +347,7 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "algorithm"
   "algorithm_container"
   "any"
+  "ascii_internal"
   "atomic_hook"
   "bad_any_cast"
   "bad_any_cast_impl"
@@ -383,6 +384,7 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "graphcycles_internal"
   "hash"
   "hash_function_defaults"
+  "hash_internal"
   "hash_policy_traits"
   "hashtable_debug"
   "hashtable_debug_hooks"
@@ -402,6 +404,7 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "node_hash_map"
   "node_hash_policy"
   "node_hash_set"
+  "numbers_internal"
   "numeric"
   "optional"
   "periodic_sampler"
-- 
2.35.1.windows.2

