From 08797647a52610ac2437c4d1b0c3af38d2704fd4 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 11 Sep 2022 10:32:19 -0400
Subject: [PATCH 6/7] default dll import for windows

---
 absl/base/config.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/absl/base/config.h b/absl/base/config.h
index e35d2e48..391da4de 100644
--- a/absl/base/config.h
+++ b/absl/base/config.h
@@ -750,10 +750,9 @@ static_assert(ABSL_INTERNAL_INLINE_NAMESPACE_STR[0] != 'h' ||
 #if defined(_MSC_VER)
 #if defined(ABSL_BUILD_DLL)
 #define ABSL_DLL __declspec(dllexport)
-#elif defined(ABSL_CONSUME_DLL)
-#define ABSL_DLL __declspec(dllimport)
 #else
-#define ABSL_DLL
+// conda-forge addition: by default, users import the definitions defined here
+#define ABSL_DLL __declspec(dllimport)
 #endif
 #else
 #define ABSL_DLL
-- 
2.33.0.windows.2

