From 9fe99781120e9e5e64b54e6ee1252f4e721bf75e Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 4 Mar 2022 12:35:49 +1100
Subject: [PATCH 8/9] add DLL export to MixingHashState::kSeed

---
 absl/hash/internal/hash.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/absl/hash/internal/hash.cc b/absl/hash/internal/hash.cc
index 11451e5..d71d41c 100644
--- a/absl/hash/internal/hash.cc
+++ b/absl/hash/internal/hash.cc
@@ -44,7 +44,7 @@ uint64_t MixingHashState::CombineLargeContiguousImpl64(
                                std::integral_constant<int, 8>{});
 }
 
-ABSL_CONST_INIT const void* const MixingHashState::kSeed = &kSeed;
+ABSL_DLL ABSL_CONST_INIT const void* const MixingHashState::kSeed = &kSeed;
 
 // The salt array used by LowLevelHash. This array is NOT the mechanism used to
 // make absl::Hash non-deterministic between program invocations.  See `Seed()`
-- 
2.35.1.windows.2

