--- absl/base/config.h	2022-06-23 14:22:47.000000000 -0400
+++ absl/base/config.h.b	2022-09-11 10:28:16.033334900 -0400
@@ -737,10 +737,9 @@
 #if defined(_MSC_VER)
 #if defined(ABSL_BUILD_DLL)
 #define ABSL_DLL __declspec(dllexport)
-#elif defined(ABSL_CONSUME_DLL)
-#define ABSL_DLL __declspec(dllimport)
 #else
-#define ABSL_DLL
+// conda-forge addition: by default, users import the definitions defined here
+#define ABSL_DLL __declspec(dllimport)
 #endif
 #else
 #define ABSL_DLL
